### ISSUE TITLE
Добавление кнопок прокрутки баров в симуляторе.

### ISSUE THREAD

**Issue by Avadacedavra** [SBProX/SBProX-Dev#1651]
Здравствуйте! Добавьте, пожалуйста, в симуляторе кнопку прокрутки бара "-1 TF", чтобы была возможность вернуть график на 1 бар назад. Будет очень удобно, если пропустил вход, вернуться обратно. http://prntscr.com/vbnalu

**Comment by Velikodushniy** [comment]
Да, поддерживаю, иногда  "-1 TF" ОЧЕНЬ нехватает. Хотелось бы такую кнопку.

**Comment by stason251187** [comment]
Поддерживаю! При тренировках часто нужно вернуться буквально на "момент" назад и проверить как и что это было. Спасибо!
